tests/test_utilities/test_sqlite_utils.py

```diff
@@ -13,7 +16,17 @@ class TestSqliteUtils(unittest.TestCase):
     def test_bulkload(self):
         if DB.exists():
             DB.unlink()
-        sqlite_bulk_load(DB, TSV, TBL_NAME, cat_cmd=["grep", "-v", "\\!"], cols=["a", "b", "c"])
+        sqlite_bulk_load(DB, TSV, TBL_NAME, cols=["a", "b", "c"])
+        con = sqlite3.connect(str(DB))
+        rows = list(con.execute(f"SELECT * FROM {TBL_NAME}"))
+        self.assertGreater(len(rows), 5)
+        self.assertIn(("MGI", "MGI:1918911", "0610005C13Rik"), rows)
+
+    def test_chunked_bulkload(self):
+        if DB.exists():
+            DB.unlink()
+        args = dict(chunksize=1000, sep="\t", comment="!", names=list("abc"))
```

Comment:
I believe for pandas this will remove *any* text that comes after a `!`, not just a line that starts with this.

Reply:
"!" is only ignoring the rest of the line but we should make sure that the first data line is not used as column names. I improved the test. https://github.com/INCATools/ontology-access-kit/pull/347